docs/documentacao_Scrum_Dr_Down.md

```diff
@@ -0,0 +1,172 @@
+# ÍNDICE
+
+- [Introdução](#introducao)
+- [Papéis na Equipe](#papeis-da-equipe)
+  - [Product Owner](#product-owner)
+  - [Arquiteto de Software](#arquiteto-de-software)
+  - [DevOps](#devops)
+  - [Scrum Master](#scrum-master)
+  - [Time de Desenvolvimento](#time-de-desenvolvimento)
+- [Ritos do Scrum](#ritos-do-scrum)
+  - [Sprint](#sprint)
+  - [Planejamento da Sprint](#planejamento-da-sprint)
+  - [Daily Meeting](#daily-meeting)
+  - [Review da Sprint](#review-da-sprint)
+  - [Retrospectiva da Sprint](#retrospectiva-da-sprint)
+
+----
+
+## Introdução
+
+Este documento tem como objetivo deixar transparente o processo Ágil utilizado pelo Time Scrum do projeto Dr. Down, descrevendo informações como papéis existentes e ritos do Scrum adotados.
+
+----
+
+## Papéis na Equipe
+
```

Comment:
como será feita a atualização dos papéis tendo em vista de que não é fixo?

Reply:
Retirei essa informação deste documento, visto que isto já está sendo informado na documentação de cada Sprint.